### PR TITLE
Update requirement of paddle version

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
Update requirement of paddle version(version>=2.6.0)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,25 +76,25 @@ DeepXDE requires one of the following backend-specific dependencies to be instal
 - TensorFlow 2.x: [TensorFlow](https://www.tensorflow.org)>=2.2.0, [TensorFlow Probability](https://www.tensorflow.org/probability)>=0.10.0
 - PyTorch: [PyTorch](https://pytorch.org)>=1.9.0
 - JAX: [JAX](https://jax.readthedocs.io), [Flax](https://flax.readthedocs.io), [Optax](https://optax.readthedocs.io)
-- PaddlePaddle: [PaddlePaddle](https://www.paddlepaddle.org.cn/en) ([develop version](https://www.paddlepaddle.org.cn/en/install/quick?docurl=/documentation/docs/en/develop/install/pip/linux-pip_en.html))
+- PaddlePaddle: [PaddlePaddle](https://www.paddlepaddle.org.cn/en)>=2.6.0 or use [develop version](https://www.paddlepaddle.org.cn/en/install/quick?docurl=/documentation/docs/en/develop/install/pip/linux-pip_en.html)
```
Comment (lululxvi) [inline README.md:79]: Remove develop version, which is not needed for most users.

**Comment by lululxvi** [comment]
Also update https://github.com/lululxvi/deepxde/blob/master/docs/user/installation.rst

File: README.md (hunk 1 above)
Comment (HydrogenSulfate) [reply README.md]: > Remove develop version, which is not needed for most users.

Removed

**Comment by HydrogenSulfate** [comment]
> Also update https://github.com/lululxvi/deepxde/blob/master/docs/user/installation.rst

Updated

**Pull request merged**